src/extensions/cp/apple/finalcutpro/inspector/audio/AudioConfiguration.lua

```diff
@@ -0,0 +1,137 @@
+--- === cp.apple.finalcutpro.inspector.audio.AudioConfiguration ===
+---
+--- The Audio Configuration section of the Audio Inspector.
+
+local require = require
+
+local log                               = require("hs.logger").new("audioConfiguration")
+
+local axutils                           = require("cp.ui.axutils")
+local Element                           = require("cp.ui.Element")
```

Comment:
Since this seems to be representing an AXScrollArea, lets just extend that directly.
```suggestion
local ScrollArea                           = require("cp.ui.ScrollArea")
```